### DISCUSSION TITLE
skywalking php channel send failed

### DISCUSSION THREAD

**Discussion by hwz9612** [apache/skywalking#10601] [Q&A]
Hello, recently, I used the skywalking php. And just as follows, encountered a problem.
`2023-03-28T02:16:48.232339Z TRACE init: skywalking_agent::request: Propagation: None
2023-03-28T02:16:48.233338Z ERROR shutdown: skywalking_agent::channel: channel send failed err=No such file or directory (os error 2)`
And I found the oap server can not receive any data. The UI don't have data.
I saw the previous issues and found you fixed the bug in version 0.3.0. But we used the version 0.3.0 and still encountered the problem.

**Comment by jmjoy** [comment]
Please try to keep php-fpm running in foreground.

https://github.com/apache/skywalking-php/pull/78
